server/controllers/utils/hashPassword.js

```diff
@@ -0,0 +1,3 @@
+const { hash } = require('bcryptjs');
```

Comment:
i think we already make a function to hash password

Reply:
@knanahassouna1, I have deleted the unused util